README.md

```diff
@@ -1,32 +1,117 @@
+    * Spring Boot [community](https://github.com/snowdrop/spring-boot-crud-booster) and [product](https://github.com/snowdrop/spring-boot-crud-booster/tree/redhat) boosters
+    * Wildfly Swarm [community](https://github.com/wildfly-swarm-openshiftio-boosters/wfswarm-rest-http-crud) and [product](https://github.com/wildfly-swarm-openshiftio-boosters/wfswarm-rest-http-crud-redhat) boosters
+* Health Check Mission
+    * Eclipse Vert.x [community](https://github.com/openshiftio-vertx-boosters/vertx-health-checks-booster) and [product](https://github.com/openshiftio-vertx-boosters/vertx-health-checks-booster-redhat) boosters
+    * Node.js [community](https://github.com/bucharest-gold/nodejs-health-check) and [product](https://github.com/bucharest-gold/nodejs-health-check-redhat) boosters
+    * Spring Boot [community](https://github.com/snowdrop/spring-boot-health-check-booster) and [product](https://github.com/snowdrop/spring-boot-health-check-booster/tree/redhat) boosters
+    * Wildfly Swarm [community](https://github.com/wildfly-swarm-openshiftio-boosters/wfswarm-health-check) and [product](https://github.com/wildfly-swarm-openshiftio-boosters/wfswarm-health-check-redhat) boosters
+* Circuit Breaker Mission
+    * Eclipse Vert.x [community](https://github.com/openshiftio-vertx-boosters/vertx-circuit-breaker-booster) and [product](https://github.com/openshiftio-vertx-boosters/vertx-circuit-breaker-booster-redhat) boosters
+    * Node.js [community](https://github.com/bucharest-gold/nodejs-circuit-breaker) and [product](https://github.com/bucharest-gold/nodejs-circuit-breaker-redhat) boosters
+    * Spring Boot [community](https://github.com/snowdrop/spring-boot-circuit-breaker-booster) and [product](https://github.com/snowdrop/spring-boot-circuit-breaker-booster/tree/redhat) boosters
+    * Wildfly Swarm [community](https://github.com/wildfly-swarm-openshiftio-boosters/wfswarm-circuit-breaker) and [product](https://github.com/wildfly-swarm-openshiftio-boosters/wfswarm-circuit-breaker-redhat) boosters
+* Secured Mission
+    * Eclipse Vert.x [community](https://github.com/openshiftio-vertx-boosters/vertx-secured-http-booster) and [product](https://github.com/openshiftio-vertx-boosters/vertx-secured-http-booster-redhat) boosters
+    * Node.js [community](https://github.com/bucharest-gold/nodejs-rest-http-secured) and [product](https://github.com/bucharest-gold/nodejs-rest-http-secured-redhat) boosters
+    * Spring Boot [community](https://github.com/snowdrop/spring-boot-http-secured-booster) and [product](https://github.com/snowdrop/spring-boot-http-secured-booster/tree/redhat) boosters
+    * Wildfly Swarm [community](https://github.com/wildfly-swarm-openshiftio-boosters/wfswarm-rest-http-secured) and [product](https://github.com/wildfly-swarm-openshiftio-boosters/wfswarm-rest-http-secured-redhat) boosters
```

Comment:
Why do we need this list?

Reply:
It is for better searching of boosters. I can transfer this info to the wiki page